src/cr/cube/dimension.py

```diff
@@ -828,8 +828,16 @@ def __len__(self):
     @lazyproperty
     def anchor_idxs(self):
         """List of int indicating the actual position of the subtotals."""
+        # TODO: this appears wrong, using an anchor (element_id) as an index (offset of
```

Comment:
I'm not sure that I understand this comment. Does it refer to how we specify the anchors, or to what this method returns? Because the method seems to return the indexes, as its name suggests. If this is in relation to how we specify things, then I don't think it's a good place for this comment (i.e. it's not responsibility of this method's scope to eventually fix it in the future).